### DISCUSSION TITLE
Fooocus 2.4.0-rc4

### DISCUSSION THREAD

**Discussion by soyv4** [lllyasviel/Fooocus#2986] [Q&A]
Good afternoon, starting today, the translation of promts is not working, what should I do?

**Comment by ILikeToasters** [comment]
Just tested and looks like a 503 error which means the service it is using (Bing) is at the moment unavailable. I just checked and they are experiencing an outage for some of their services including API.

**Comment by soyv4** [comment]
Okay, thanks for confirming the error, we are waiting for corrections
